packages/core/src/App/Components/Layout/Header/account-info-wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Popover } from '@deriv/components';
+
+type TAccountInfoWrapper = {
+    is_disabled?: boolean;
+    disabled_message?: string;
+};
+
+const AccountInfoWrapper = ({
+    is_disabled,
+    disabled_message,
+    children,
+}: React.PropsWithChildren<TAccountInfoWrapper>) =>
+    is_disabled && disabled_message ? (
+        <Popover alignment='left' message={disabled_message} zIndex={'99999'}>
```

Comment:
what happens if you let the zIndex to default (1)? Will it still work?

Reply:
@aizad-deriv it will work I suppose (I can't test it because qa29 is rebuildng now), but I'm not sure I need to change it because value 99999 was tested 3 years ago and it works for now, if I change it it can lead to unexpected consequences. What do you think? If you still think I need to decrease this value to 1 then I'll do this :) Let me know

Reply:
Maybe just try it first but if it didn't then just leave it as it is 👍🏼

Reply:
@aizad-deriv First image with zIndex === 1, second with zIndex === 9999. So we need to leave 99999

<img width="424" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/9b825445-b27b-4d57-a80a-5b6e548920cc">
<img width="565" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/185423cd-ec63-4fdd-bf45-c0b22c418b53">